README.md

```diff
@@ -135,29 +135,29 @@ There are some optional dependencies that are required for specific functionalit
 which will not be installed automatically by conda or pip.
 See [Dependencies](#dependencies) for details on installing optional dependencies.
 
-Note that as of v2.2, `pyuvdata` is only supported on python 3.7+.
```

Comment:
this is a big change. I think I would be in favor of making a version for this.

Reply:
note that this is really only dropping python 3.9. We already dropped 3.8 back in v2.4.2, we just forgot to update this line in the README. We did note it in the changelog.

I agree about the new version. We need one once this is merged for pyuvsim's sake anyway.

Reply:
oh, I also just realized that I didn't update the version number here. I'm going to restructure this a little to make it easier to keep it up to date.

Reply:
I moved it into the list of required dependencies. I think it'll be more likely to be kept up to date if it's with the other dependencies so we see it when we're changing the other ones.

Reply:
sounds good thank you!